tests/modules/bowtie2/align/test.yml

```diff
@@ -1,83 +1,21 @@
-- name: bowtie2 align single-end
-  command: nextflow run ./tests/modules/bowtie2/align -entry test_bowtie2_align_single_end -c ./tests/config/nextflow.config -c ./tests/modules/bowtie2/align/nextflow.config
+- name: bowtie2 align test_bowtie2_align_single_end
+  command: nextflow run tests/modules/bowtie2/align -entry test_bowtie2_align_single_end -c tests/config/nextflow.config
   tags:
-    - bowtie2
     - bowtie2/align
-  files:
-    - path: ./output/bowtie2/test.bam
-    - path: ./output/bowtie2/test.bowtie2.log
-    - path: ./output/bowtie2/bowtie2/genome.3.bt2
-      md5sum: 4ed93abba181d8dfab2e303e33114777
-    - path: ./output/bowtie2/bowtie2/genome.2.bt2
-      md5sum: 47b153cd1319abc88dda532462651fcf
-    - path: ./output/bowtie2/bowtie2/genome.1.bt2
-      md5sum: cbe3d0bbea55bc57c99b4bfa25b5fbdf
-    - path: ./output/bowtie2/bowtie2/genome.4.bt2
-      md5sum: c25be5f8b0378abf7a58c8a880b87626
-    - path: ./output/bowtie2/bowtie2/genome.rev.1.bt2
-      md5sum: 52be6950579598a990570fbcf5372184
-    - path: ./output/bowtie2/bowtie2/genome.rev.2.bt2
-      md5sum: e3b4ef343dea4dd571642010a7d09597
-
-- name: bowtie2 align paired-end
-  command: nextflow run ./tests/modules/bowtie2/align -entry test_bowtie2_align_paired_end -c ./tests/config/nextflow.config -c ./tests/modules/bowtie2/align/nextflow.config
-  tags:
     - bowtie2
-    - bowtie2/align
   files:
-    - path: ./output/bowtie2/test.bam
-    - path: ./output/bowtie2/test.bowtie2.log
-    - path: ./output/bowtie2/bowtie2/genome.3.bt2
-      md5sum: 4ed93abba181d8dfab2e303e33114777
-    - path: ./output/bowtie2/bowtie2/genome.2.bt2
-      md5sum: 47b153cd1319abc88dda532462651fcf
-    - path: ./output/bowtie2/bowtie2/genome.1.bt2
-      md5sum: cbe3d0bbea55bc57c99b4bfa25b5fbdf
-    - path: ./output/bowtie2/bowtie2/genome.4.bt2
-      md5sum: c25be5f8b0378abf7a58c8a880b87626
-    - path: ./output/bowtie2/bowtie2/genome.rev.1.bt2
-      md5sum: 52be6950579598a990570fbcf5372184
-    - path: ./output/bowtie2/bowtie2/genome.rev.2.bt2
-      md5sum: e3b4ef343dea4dd571642010a7d09597
```

Comment:
These have been all deleted because the BAM won't be present without the index files? Is there any harm in leaving them in as a sanity check?

Reply:
This file was auto-generated by `create-test-yaml`, so I've no vested interest in the content

Reply:
I also have no idea why the index files have ever been included in the output from `bowtie2 align`

Reply:
Again, no idea what happened here 😏 Does `/output/bowtie2/bowtie2/` exist when you run `pytest`?

Reply:
```
.
├── 40
│   └── fde8dfbf23b62799ec1c8fe1edc520
│       ├── bowtie2 -> /tmp/pytest_workflow_um_6gy_g/bowtie2_align_test_bowtie2_align_single_end/work/43/d87f151078c05ee619a09f9adaae31/bowtie2
│       ├── test.bam
│       ├── test.bowtie2.log
│       ├── test_1.fastq.gz -> /tmp/pytest_workflow_um_6gy_g/bowtie2_align_test_bowtie2_align_single_end/work/stage/07/142d3883569478136b1e5e6536ffbd/test_1.fastq.gz
│       └── versions.yml
├── 43
│   └── d87f151078c05ee619a09f9adaae31
│       ├── bowtie2
│       │   ├── genome.1.bt2
│       │   ├── genome.2.bt2
│       │   ├── genome.3.bt2
│       │   ├── genome.4.bt2
│       │   ├── genome.rev.1.bt2
│       │   └── genome.rev.2.bt2
│       ├── genome.fasta -> /tmp/pytest_workflow_um_6gy_g/bowtie2_align_test_bowtie2_align_single_end/work/stage/7a/1e46d5cd1dad747bdec96651090752/genome.fasta
│       └── versions.yml
└── stage
    ├── 07
    │   └── 142d3883569478136b1e5e6536ffbd
    │       └── test_1.fastq.gz
    └── 7a
        └── 1e46d5cd1dad747bdec96651090752
            └── genome.fasta
```

Reply:
Sorry, in the output directory not the work directory.

Reply:
```
output/
└── bowtie2
    ├── bowtie2 -> /tmp/pytest_workflow_82vq1t4y/bowtie2_align_test_bowtie2_align_single_end/work/8b/b0d4b9040c8329f51428d5e92b6812/bowtie2
    │   ├── genome.1.bt2
    │   ├── genome.2.bt2
    │   ├── genome.3.bt2
    │   ├── genome.4.bt2
    │   ├── genome.rev.1.bt2
    │   └── genome.rev.2.bt2
    ├── test.bam -> /tmp/pytest_workflow_82vq1t4y/bowtie2_align_test_bowtie2_align_single_end/work/e7/61e8210898ae529cbed8912da0512a/test.bam
    ├── test.bowtie2.log -> /tmp/pytest_workflow_82vq1t4y/bowtie2_align_test_bowtie2_align_single_end/work/e7/61e8210898ae529cbed8912da0512a/test.bowtie2.log
    └── versions.yml -> /tmp/pytest_workflow_82vq1t4y/bowtie2_align_test_bowtie2_align_single_end/work/e7/61e8210898ae529cbed8912da0512a/versions.yml
```